physics/mass_energy_equivalence.py

```diff
@@ -0,0 +1,42 @@
+"""
+Mass energy Equivalene
+Reference: https://en.wikipedia.org/wiki/Mass%E2%80%93energy_equivalence
+"""
+import math as m
+
+""" Function to calculate energy(in joules) equivalent of a given
+mass(in kilograms) using Einstein's equation E=mc^2"""
```

Comment:
Please combine these two docstrings and provide a brief explanation of the equation